tests/TestCase/Command/BootstrapTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Cake\Codeception\Test\TestCase\Command;
+
+use PHPUnit_Framework_TestCase as TestCase;
+
+class BootstrapTest extends TestCase
+{
+
+    public function testHelpersCreatedInTestSuite()
+    {
+        $testSuiteDir = TEST_APP_ROOT . 'src' . DS . 'TestSuite' . DS . 'Codeception' . DS;
+        $this->assertTrue(is_dir($testSuiteDir), 'src/TestSuite/Codeception directory must be auto-created');
+        $this->assertTrue(file_exists($testSuiteDir . 'AcceptanceHelper.php'), 'AcceptanceHelper must be auto-created');
+        $this->assertTrue(file_exists($testSuiteDir . 'FunctionalHelper.php'), 'FunctionalHelper must be auto-created');
+        $this->assertTrue(file_exists($testSuiteDir . 'UnitHelper.php'), 'UnitHelper must be auto-created');
+    }
+
+    public function testSuitesCreatedInTests()
+    {
+        $testsDir = TEST_APP_ROOT . 'tests' . DS;
+
+        $acceptanceDir = $testsDir . 'acceptance' . DS;
+        $this->assertTrue(is_dir($acceptanceDir), 'Acceptance test suite directory must be auto-created');
+        $this->assertTrue(file_exists($acceptanceDir . 'AcceptanceTester.php'), 'AcceptanceTester must be auto-generated');
+        $this->assertTrue(file_exists($acceptanceDir . 'bootstrap.php'), 'Acceptance bootstrap.php must be auto-created');
+
+        $functionalDir = $testsDir . 'functional' . DS;
+        $this->assertTrue(is_dir($functionalDir), 'Functional test suite directory must be auto-created');
+        $this->assertTrue(file_exists($functionalDir . 'FunctionalTester.php'), 'FunctionalTester must be auto-generated');
+        $this->assertTrue(file_exists($functionalDir . 'bootstrap.php'), 'Functional bootstrap.php must be auto-created');
+
+        $unitDir = $testsDir . 'unit' . DS;
+        $this->assertTrue(is_dir($unitDir), 'Unit test suite directory must be auto-created');
+        $this->assertTrue(file_exists($unitDir . 'UnitTester.php'), 'UnitTester must be auto-generated');
+        $this->assertTrue(file_exists($unitDir . 'bootstrap.php'), 'Unit bootstrap.php must be auto-created');
```

Comment:
It also needs to assert more files' contents (custom configuration stuff to run with cake out of the box).